tests/StatsdClient.Tests/StatsdClient.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
Do we need to drop this? I think we still want the ability to run tests on .NET Framework.

Reply:
More than happy to add it back in.  I'm doing this on linux so haven't got net461 to actually target against